### PR TITLE
create mem store keys

### PR DESCRIPTION
**Pull request by 0xmuralik**
# Related Github tickets

- a list of github issues that are relevant for this PR
- Closes: #811 

# Background

_A short paragraph on test what exactly this PR solves._

# Testing completed

- [ ] _a list of actions that you've performed to ensure that this PR works as intended_
- [ ] _99% of the times you should have an item: "wrote tests"_

# Breaking changes

- [ ] I have checked my code for breaking changes
- [ ] If there are breaking changes, there is a supporting migration.

### CODE REVIEW

File: app/app.go [hunk 1]
```diff
@@ -184,7 +184,6 @@ var (
 		ibctm.AppModuleBasic{},
 		ica.AppModuleBasic{},
 		ibcfee.AppModuleBasic{},
-		consensus.AppModuleBasic{},
```
Comment (0xmuralik) [inline app/app.go:187]: registered twice

**Review by taariq — APPROVED** [review]

File: app/app.go (hunk 1 above)
Comment (taariq) [reply app/app.go:187]: Is that a problem?

Comment (0xmuralik) [reply app/app.go:187]: https://github.com/palomachain/paloma/blob/f2e6d757e296e89c09e4456286b49bb06a71524c/app/app.go#L181
https://github.com/palomachain/paloma/blob/f2e6d757e296e89c09e4456286b49bb06a71524c/app/app.go#L187
It's not required to add it twice, right?

Comment (taariq) [reply app/app.go:187]: No. It's not. Let's remove one so we keep only one.

**Pull request merged**